wiedesign/GarbageCollectorRobot
# Install Prettier and ESLint

## Desciption
closes #19 

Installed the `Prettier` and `ESLint` npm packages and wrote npm commands to run the corresponding CLI commands to check project files:
`npm run lint:prettier`
`npm run lint:eslint`

These commands should be run before committing any changes so you can see which files that need to be reformatted.

I also created a workspace file for the project folder to allow for extension recommendations. This will prompt VS Code users to install these extensions and that should help you format your files moving forward. On VS Code, hitting save should automatically format your files.

## Acceptance Criteria
- [x] Install Prettier
- [x] Install ESLint

## Test Instructions
I think the best way to test this is to checkout this branch and write some code to see if the linter and formatter works :)
